experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusSerializer.test.ts

```diff
@@ -164,7 +164,7 @@ describe('PrometheusSerializer', () => {
   });
 
   describe('serialize an instrumentation metrics', () => {
-    describe('Singular', () => {
```

Comment:
Maybe we need to add a new section for non-monotonic DataPointType.SUM and DataPointType.Gauge?

Reply:
Great idea. :slightly_smiling_face: Non-monotonic sum and gauge tests are added as of [1a442db](https://github.com/open-telemetry/opentelemetry-js/pull/3079/commits/1a442db41ca543647517c1ab20ebe2d3bb310f55). :slightly_smiling_face: